docs/release-notes/2.15.0.md

```diff
@@ -0,0 +1,95 @@
+* [PMM-6671](https://jira.percona.com/browse/PMM-6671): Ability to add HAProxy instances (UI)
+* [PMM-6640](https://jira.percona.com/browse/PMM-6640): Ability to add HAProxy services (API) (Technical Preview)
+* [PMM-5778](https://jira.percona.com/browse/PMM-5778): Check connection response format when adding external agents (UI)
+* [PMM-4172](https://jira.percona.com/browse/PMM-4172): Ability to execute `pt-mysql-summary` for MySQL service (API)
+
+
+
+## Improvements
+
+* [PMM-6693](https://jira.percona.com/browse/PMM-6693): pmm-managed denies request when using API keys even if the role set is admin -- The PMM API will now accept requests not only with username/password but also with API tokens generated in Grafana. 
+* [PMM-7487](https://jira.percona.com/browse/PMM-7487): Check connection to HAProxy services when adding them for monitoring (UI) (Thanks to Jiří Čtvrtka for reporting this issue)
+* [PMM-7476](https://jira.percona.com/browse/PMM-7476): Improve `pmm-admin --help` output, highlight required fields for external services
+* [PMM-7473](https://jira.percona.com/browse/PMM-7473): Improve descriptions of `pmm-admin` flags and required fields for external services
+* [PMM-7471](https://jira.percona.com/browse/PMM-7471): Update VictoriaMetrics 1.52.0 to 1.53.1 (See [VictoriaMetrics 1.53.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.53.0) and [VictoriaMetrics 1.53.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.53.1).)
+* [PMM-7433](https://jira.percona.com/browse/PMM-7433): Integrated Alerting: Easier-to-read rule details in Alert Rules list
+* [PMM-7422](https://jira.percona.com/browse/PMM-7422): Security Threat Tool: View all active and silenced alerts
+* [PMM-7331](https://jira.percona.com/browse/PMM-7331): Security Threat Tool: Failed checks have 'Read more' links with helpful content
+* [PMM-7329](https://jira.percona.com/browse/PMM-7329): Security Threat Tool: Hide check results tab if STT is disabled
+* [PMM-7328](https://jira.percona.com/browse/PMM-7328): Security Threat Tool: Download and run checks when activated, immediately, repeating every 24 hours thereafter (Previously, downloading and running new checks happened every 24 hours but the cycle didn't begin when STT was activated.)
+* [PMM-7259](https://jira.percona.com/browse/PMM-7259): Integrated Alerting: 'Alerting is disabled' message gives hint to enable it
+* [PMM-7257](https://jira.percona.com/browse/PMM-7257): Integrated Alerting: Easier-to-read rule details in Alert Rules list (API to expose Alert Rule's expression and real values)
+* [PMM-7240](https://jira.percona.com/browse/PMM-7240): DBaaS: Change from Dashboard to Grafana Page -- We changed the DBaaS page from a Grafana Dashboard to a Grafana Page to be better aligned with the DBaaS enable/disable status and avoid confusion when DBaaS is disabled.
```

Comment:
I'm concerned we're blurring the lines between features that are GA and what's still very much a tech preview...maybe nothing to do but these are still not GA Features so does mixing them in with GA stuff send the wrong impression?

Reply:
This is for the last iteration when we'll do re-ordering.
 we need to group issue  by features  (released ones) then all Tech preview related